djangocms_navigation/admin.py

```diff
@@ -165,23 +176,30 @@ def change_view(
         extra_context = extra_context or {}
         if menu_content_id:
             request.menu_content_id = menu_content_id
-            if self._versioning_enabled:
-                menu_content = get_object_or_404(
-                    MenuContent._base_manager, id=menu_content_id
-                )
-                version = Version.objects.get_for_content(menu_content)
-                try:
-                    version.check_modify(request.user)
-                except ConditionFailed as error:
-                    messages.error(request, str(error))
-                    return HttpResponseRedirect(version_list_url(menu_content))
-                # purge menu cache
-                purge_menu_cache(site_id=menu_content.menu.site_id)
-            extra_context["list_url"] = reverse(
-                "admin:djangocms_navigation_menuitem_list",
-                kwargs={"menu_content_id": menu_content_id},
+
+        if self._versioning_enabled:
+            menu_content = get_object_or_404(
+                self.MenuModel._base_manager, id=menu_content_id
```

Comment:
I can't see where self.MenuModel is defined. Is this defined anywhere?

Reply:
@jonathan-s I couldn't see where this was used, it's possibly left over from changes in the other PR for making the models reusable: self.MenuModel